code/modules/research/designs/misc_designs.dm

```diff
@@ -469,3 +474,28 @@
 	build_path = /obj/item/reactive_armour_shell
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING
+
+/////////////////////////////////////////
+////////////Meteor///////////////////////
+/////////////////////////////////////////
+
+/datum/design/meteor_defence
+	name = "Meteor Defence"
+	desc = "A blue print of a early model of the Meteor defence turret."
+	id = "meteor_defence"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 150000, MAT_GLASS = 150000, MAT_SILVER = 7500, MAT_GOLD = 10000, MAT_TITANIUM = 35000) //Station goal

```

Comment:
Making things cost a shitload of a certain type of materials is not how you should be balancing these things. Make them cost more material types, but less metal/glass, why does a single shield need as much metal as to build a small room?

Reply:
So we dont spam them around ever shift, that way cargo is main force behind buying them. I did the math and its around 20 metal and glass when maxed well others are less then 12~ each